### PR TITLE
Implementation of flavor feature based on new master

### PR DESCRIPTION
**Pull request by DblK**
It implements a new parameter "flavor" which is by default "sdk" that
allow to choose the flavor of the build (useful for dev and production
application).

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -185,6 +186,7 @@ NwBuilder.prototype.resolveLatestVersion = function () {
 
 NwBuilder.prototype.checkVersion = function () {
     var version = this.options.version,
+        flavor = semver.satisfies(version, '<0.12.3') ? 'sdk' : this.options.flavor,
```
Comment (linquize) [inline lib/index.js]: Should move this line above `var getVersionFromManifest = function(){`

File: lib/index.js [hunk 2]
```diff
@@ -186,8 +186,8 @@ NwBuilder.prototype.resolveLatestVersion = function () {
 
 NwBuilder.prototype.checkVersion = function () {
     var version = this.options.version,
-        flavor = semver.satisfies(version, '<0.12.3') ? 'sdk' : this.options.flavor,
         self = this;
+    var flavor = semver.satisfies(version, '<0.12.3') ? 'sdk' : this.options.flavor;
```
Comment (linquize) [inline lib/index.js]: OOPS, this is not the right position. 

Should move to the line above `var getVersionFromManifest = function(){`

**Comment by gaving** [comment]
:+1: (wanting this for production builds)

**Comment by adam-lynch** [comment]
Hey, thanks for this! Please add / update the tests though 😀

**Comment by bhavyaw** [comment]
@adam-lynch @DblK Any chance it will get merged ?

Is there any other we can opt for a specific flavor rather default sdk one.

**Comment by chrisneal** [comment]
Any movement on this?

**Comment by bhavyaw** [comment]
I don't know when this will be merged into repo but i was able to figure out which file we need to make the changes so we can download the **normal flavor**.

Here's what you need to do :

1. Open file `Versions.js` ( Capital V ) 
2. Search for method `generatePlatformUrls`
3. Replace String in that method from '/nwjs-sdk-v' to '/nwjs-v'


downloadingsdkflavor.

**Pull request closed without merging**